src/pages/open-source.js

```diff
@@ -21,6 +27,14 @@ const OpenSource = ({ data }) => {
     <Layout>
       <Head page={{ title, slug, seoTitle, seoMetaDescription }} />
       {/* <CaseStudyPreview caseStudy={} /> */}
+      <TalksSection
+        icon={talksSectionImage}
+        title={talksSectionTitle}
+        talks={talksSectionTalks}
+        ctaText={talkSectionCtaText}
+        ctaLink={talksSectionCtaLink}
+      />
+      <p>open source page</p>
```

Comment:
Please rebase onto the latest `open-source-page` branch, to avoid putting things like this back in :pray:

Reply:
Done.